downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1562 @@
+
+- All client initializations return an `email:Error` when an error occurs during the initialization.
+
+###### Service and Listener Related Changes
+
+- All listener initializations return an `email:Error` when an error occurs during the initialization.
+
+- The `email:Listener` is split into the `email:PopListener` and `email:ImapListener`. Therefore, the `protocol` field is removed from the new protocol-specific listeners. The `email:PopConfig` or `email:ImapConfig` that were used as fields for the `email:Listener` are not required for the new API implementation. Protocol configuration related fields are made parts of the new listeners.
+
+- The `resource` functions are changed to `remote` functions in the new listener APIs.
+
+- The `service` name is given as a string with the new Ballerina language changes.
+
+- The `onMessage` method of the `email:Listener` (i.e., `new email:PopListener` and `email:ImapListener`) are changed to `onEmailMessage`.
+
+- The `pollingInterval` field of the `email:Listener` is changed to `pollingIntervalInMillis` in the new listener APIs. That makes it consistent across other Ballerina modules, which are time durations configured in milliseconds.
+
+- A sample POP3 listener is given below.
+
+    **New Syntax:**
+
+    ```ballerina
+    listener email:PopListener emailListener = new ({
+        host: "pop.email.com",
+        username: "reader@email.com",
+        password: "pass456",
+        pollingIntervalInMillis: 2000,
+        port: 995
+    });
+
+    service "emailObserver" on emailListener {
+
+    remote function onEmailMessage(email:Message emailMessage) {
+
+        }
+
+    remote function onError(email:Error emailError) {
+
+        }
+    }
+    ```
+
+##### WebSub Module Improvements
+
+###### Hub-Related Changes
+
+- Default implementation for the `websub:Hub` has been removed from the module.
+- API specification for the WebSub Hub is moved to the [`websubhub` module](#websubhub).
+
+##### Publisher-Related Changes
+
+- The implementation related to the `websub:PublisherClient` is moved to the `websubhub` module.
+
+##### Subscriber-Related Changes
+
+- The two new configurations below are introduced to the `@websub:SubscriberServiceConfiguration`.
+  - `accept` - The expected media type
+  - `acceptLanguage` - The expected language type
+- API specification for the `@websub:SubscriberService` is updated with the changes below.
+  - The `onIntentVerification` and `onNotification` functions are removed.
+  - The `onSubscriptionVerification`, `onEventNotification`, and `onSubscriptionValidationDenied` functions are introduced.
+- The updated `websub:SubscriberService` sample implementation is as follows.
+
+    ```ballerina
+    listener websub:Listener testListener = new (9090);
+
+    @websub:SubscriberServiceConfig {
+        path: "/websub",
+        subscribeOnStartUp: false,
```

Comment:
@ayeshLK Let's remove `path` and `subscriberOnStartup` configs. I am not sure if path is a mandatory config though as discussed we need to remove it in the next release.

Reply:
```suggestion
```